src/statmodels.jl

```diff
@@ -259,10 +271,9 @@ In this formula, ``L`` is the likelihood of the model, ``L0`` that of the null m
 of the model (as returned by [`dof`](@ref)).
 """
 function adjr2(obj::StatisticalModel, variant::Symbol)
-    ll = -deviance(obj)/2
-    ll0 = -nulldeviance(obj)/2
+    ll = likelihood(obj)
```

Comment:
AFAICT you want `loglikelihood` here. The fact it wasn't caught by testing is a bit worrying: isn't this covered by GLM's tests?

Reply:
Sorry. You're right. I hadn't tested the adjusted R².

Reply:
OK. Do you confirm it works know?

Reply:
Yes, sir!